server/models/post.js

```diff
@@ -18,4 +21,4 @@ const PostSchema = mongoose.Schema(
 
 const Post = mongoose.model('Post', PostSchema);
 
-module.exports = {Post};
+module.exports = Post;
```

Comment:
Just flagging this to make sure you're aware of the distinction of exporting an object vs exporting the model itself. I don't think there's a huge difference in these two, but I am curious why you decided to change this. Nothing wrong with it, just curious

Reply:
When we were exporting the entire model there was an error and I don't know why but this made it work. The error was that the export didn't exist.